### PR TITLE
Dump out stats for each iface

### PR DESCRIPTION
**Pull request by bowei**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -127,6 +127,13 @@ while true; do
     fi
   done
 
+  # ethtool for every interface
+  for iface in $(ip link | awk '/^[0-9]+: / {print $2}' | sed 's/:$//' | sed 's/@.*$//'); do
+    f=ethtool_${iface}
+    c=$(ethtool -S ${iface})
```
Comment (bobbypage) [inline run.sh]: is this ok running within container or will being in network namespace be ok?

Comment (karan) [reply run.sh]: with host network it should be fine

Comment (bowei) [reply run.sh]: I assuming dumping the host side of the veth is sufficient
We can poke into each NS for completeness

**Comment by bobbypage** [comment]
LGTM.

**Pull request merged**